keel-core/src/main/kotlin/com/netflix/spinnaker/keel/persistence/memory/InMemoryDeliveryConfigRepository.kt

```diff
@@ -221,6 +222,15 @@ class InMemoryDeliveryConfigRepository(
       .map { name -> configs[name] ?: error("No delivery config named $name") }
   }
 
+  // TODO[GY]: this does not take into account puased applications, as we can't wire it here.
```

Comment:
As discussed in chat, if #1069 gets merged, then you can take advantage of the changes in the following file and implement this directly in `InMemoryCombinedRepository`, where you'll have access to both delivery configs and pauses.
https://github.com/spinnaker/keel/pull/1069/files#diff-0501cf0c59a58dc72aee2333bb484a56

Reply:
That would be great! I will make a note to myself to change it.